tests/TypeProviderHelper.php

```diff
@@ -0,0 +1,241 @@
+<?php
+
+namespace WpOrg\Requests\Tests;
+
+use ArrayIterator;
+use EmptyIterator;
+use stdClass;
+use WpOrg\Requests\Tests\Fixtures\ArrayAccessibleObject;
+use WpOrg\Requests\Tests\Fixtures\StringableObject;
+
+final class TypeProviderHelper {
+
+	/** Keys of all type entries representing null. */
```

Comment:
We should add type information for these constants as well.
```suggestion
	/**
	 * Keys of all type entries representing null.
	 *
	 * @var array<string>
	 */
```

Reply:
For this and the other remark: absolutely valid point, I was just cheating/taking a shortcut as this is test only code, but you're right, let's lead by example and do it properly.